### PR TITLE
42/textual enhancement

### PR DESCRIPTION
**Pull request by SaraTorrey**
This PR addresses issue #42 to improve the text of the docs.

### CODE REVIEW

File: content/legacy_docs_af/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 = Legacy Axon Framework Documentation
 
-NOTE: The Axon Documentation contain documentation for Axon Framework 4.8 onwards. The menu on the left links to respective documentations for previous versions.
\ No newline at end of file
+NOTE: The Axon Documentation contains documentation for Axon Framework 4.8 onwards. The menu on the left links to the documentation for previous versions.
```
Comment (smcvb) [inline content/legacy_docs_af/modules/ROOT/pages/index.adoc:3]:
```suggestion
NOTE: The Axon Documentation contains documentation for Axon Framework 4.10 onwards. The menu on the left links to the documentation for previous versions.
```

File: content/legacy_docs_as/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,3 +1,3 @@
 = Legacy Axon Server Documentation
 
-NOTE: The Axon Documentation contain documentation for Axon Server 4.8 onwards. The menu on the left links to respective documentations for previous versions.
\ No newline at end of file
+NOTE: The Axon Documentation contains documentation for Axon Server 4.8 onwards. The menu on the left links to the documentation for previous versions.
```
Comment (smcvb) [inline content/legacy_docs_as/modules/ROOT/pages/index.adoc:3]:
```suggestion
NOTE: The Axon Documentation contains documentation for Axon Server 2024.1 onwards. The menu on the left links to the documentation for previous versions.
```

**Review by smcvb — CHANGES_REQUESTED** [review]
Although it's not a change you made, the referred to version on the legacy docs pages is wrong.

**Comment by SaraTorrey** [comment]
@smcvb made the requested changes. Thank you for catching those. Let me know if I need to change anything else.

**Comment by CodeDrivenMitch** [comment]
Since the merge was complex, I did it for you @SaraTorrey, hope you don't mind

**Review by smcvb — APPROVED** [review]
My concerns have been addressed, hence I'm approving this pull request.

**Pull request merged**